users/urls_api.py

```diff
@@ -0,0 +1,9 @@
+from django.urls import path
+from . import views_api
+
+urlpatterns = [
+    path('users/', views_api.UsersView.as_view()),
+    path('users/<int:pk>/', views_api.UserDetailView.as_view()),
```

Comment:
This should be `users/me` endpoint. There is no point of sending the id since this will work for only authenticated user. View needs to change accordingly.